docs/src/main/asciidoc/WildFly_and_WildFly_Preview.adoc

```diff
@@ -62,17 +62,100 @@ This feature pack is the WildFly Preview analogue to main WildFly's ``wildfly@ma
+Often, but not always, a key difference between standard WildFly and WildFly Preview is support for different versions of Jakarta EE. The WildFly project uses WildFly Preview to showcase full or partial support for new versions of Jakarta EE, including releases like EE 11 that are not yet released for general availability.
 
 Note that formally certifying WildFly Preview as compatible implementation of Jakarta EE is not a priority
 for the WildFly project and may not happen at the time of a release, or ever. Users interested in formal EE
 compliance of WildFly Preview should check the https://github.com/wildfly/certifications/tree/EE10[WildFly Certifications repository].
 
+[wildfly-preview-ee11]
+=== EE 11 Support in WildFly Preview
+
+The 32 release introduces a significant inflection in how we are using WildFly Preview. Beginning with this release we are starting to use WildFly Preview to provide a look at what we're doing for Jakarta EE 11 support.  EE 11 is not yet GA, and standard WildFly won't support EE 11 before the WildFly 34 release, at earliest. But there are milestone, release candidate and final releases of many EE 11 specs and implementations available, so we decided to provide those in WildFly Preview. This means for a number of EE APIs, WildFly Preview no long provides an EE 10 compatible implementation.
+
+However, for a number of specifications that are planning changes for EE 11 we are still offering the EE 10 variant. In future releases we'll shift those to the EE 11 variants.
+
+The following table lists the various Jakarta technologies offered by WildFly Preview 32, along with information about which EE platform version the specification relates to. Note that a number of Jakarta specifications are unchanged between EE 10 and EE 11, while other EE technologies that WildFly offers are not part of EE 11.
+
+[cols=",,",options="header"]
+|=======================================================================
+|Jakarta EE Technology |Specification Version| EE Version
+
+|Jakarta Activation| 2.1 |10 & 11
+
+|Jakarta Annotations| 3.0.0 |11
+
+|Jakarta Authentication| 3.0 |10
+
+|Jakarta Authorization| 3.0.0-M2 |11
+
+|Jakarta Batch| 2.1 |10 & 11
+
+|Jakarta Concurrency| 3.1.0-M1 |11
+
+|Jakarta Connectors| 2.1 |10 & 11
+
+|Jakarta Contexts and Dependency Injection| 4.1.0 |11
+
+|Jakarta Debugging Support for Other Languages| 2.0 |10 & 11
+
+|Jakarta Dependency Injection| 2.0 |10 & 11
+
+|Jakarta Enterprise Beans| 4.0 |10 & 11
+
+|Jakarta Enterprise Web Services| 2.0 |10 xref:note1[^1^]
+
+|Jakarta Expression Language| 6.0.0 |11
+
+|Jakarta Faces| 4.1.0-M1 |11
+
+|Jakarta Interceptors| 2.2.0 |11
+
+|Jakarta JSON Binding| 3.0 |10 & 11
+
+|Jakarta JSON Processing| 2.1 |10 & 11
+
+|Jakarta Mail| 2.1 |10 & 11
+
+|Jakarta Messaging| 3.1 |10 & 11
+
+| Jakarta MVC
+(_preview stability only_)| 2.1| N/A xref:note2[^2^]
+
+|Jakarta Persistence| 3.2.0-M2 |11
+
+|Jakarta RESTful Web Services| 3.1 |10
```

Comment:
Consider changing to `4.0` https://jakarta.ee/specifications/restful-ws/4.0/ when possible.